storage_service/locations/models/package.py

```diff
@@ -783,7 +785,10 @@ def check_fixity(self, delete_after=True):
         if self.is_compressed:
             # bagit can't deal with compressed files, so extract before
             # starting the fixity check.
-            path, temp_dir = self.extract_file()
+            try:
+                 path, temp_dir = self.extract_file()
+            except StorageException:
+                 return (False, [], 'Error extracting file')
```

Comment:
The `BagValidationError` does have a details attribute which is a list - see https://github.com/LibraryOfCongress/bagit-python/blob/master/bagit.py#L558-L566  However, I think for `None`-errors there isn't a list of different validation errors that failed (because no validation was run), and only the `message` is populated.